apps/src/templates/progress/ProgressLegend.jsx

```diff
@@ -177,26 +187,88 @@ export default class ProgressLegend extends Component {
             <TD>N/A</TD>
           </tr>
           <tr style={styles.subsequentRow}>
-            <TD style={styles.rightBorder}>{i18n.activity()}</TD>
+            <TD style={styles.rightBorder} rowSpan={secondRowRowSpan}>
+              {i18n.activity()}
+            </TD>
             <TD>
-              <div style={styles.iconAndText}>
-                <FontAwesome icon="scissors" style={styles.icon} />
-                {i18n.unplugged()}
-              </div>
+              <table>
```

Comment:
nit: can we get away with having this be 2 div elements rather than a nested table element? i'm wondering if something like this would have the same result with less code:

```
<TD>
  <div style={styles.iconAndText}>
    <FontAwesome icon="scissors" style={styles.icon} />
    {i18n.unplugged()}
  </div>
  <div style={styles.iconAndText}>
    <FontAwesome
      icon="flag-checkered"
      style={styles.icon}
    />
    {i18n.stageExtras()}
  </div>
</TD>
```

Reply:
Thanks for the suggestion! I was able to get something like this to work